src/molecules/todoItem/TodoItem.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+//Styles
+import "./TodoItem.css"
+
+const TodoItem = ({todoInfo, onTodoDelete }) => {
+
+    const {id, text} = todoInfo;
+
+    const _onTodoDelete = () => {
```

Comment:
_onTodoDelete <> onTodoDelete, Do not used _ as prefix for methods, Please updated wherever applicable.